cfgov/prepaid_agreements/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.conf.urls import url
+
+from prepaid_agreements.views import detail, index
+
+
+urlpatterns = [
+    url(r'^$', index, name='index'),
```

Comment:
The names `index` and `detail` here either should be changed to `prepaid_agreements_index` and `prepaid_agreements_detail` OR this urls.py file should be included [in the main file](https://github.com/cfpb/cfgov-refresh/pull/5207/files#diff-e45111e649c8f0371d5367b7351f1dd8R238) with a [namespace](https://docs.djangoproject.com/en/1.11/topics/http/urls/#url-namespaces) ([example](https://github.com/cfpb/cfgov-refresh/blob/48c3dfe6733e25dd584bc606f4ce73d8b9f93d0b/cfgov/cfgov/urls.py#L231)). Otherwise this will be defining the one single "index" URL for the entire site.

Reply:
Thanks for pointing this out! Fixed in https://github.com/cfpb/cfgov-refresh/commit/418c21c8291dad2f5ba17c285c4d1a6b838aab4c